### DISCUSSION TITLE
shamiko-release100对android10的MIUI12.0.11无法隐藏bl锁

### DISCUSSION THREAD

**Discussion by cok3307788** [LSPosed/LSPosed#1701] [Q&A] [answered]
红米note9 pro
android10
MIUI12.0.11稳定版
magisk 24102 +shamiko 0.4.3
我确定已经正确使用了shamiko，momo已经检测不到magisk和su
在开发者选项里面能看到设备未解锁，但是用momo会提示bl已解锁

**Comment by yujincheng08** [comment, marked as answer]
shamiko 有说过会隐藏 bl 锁？

**Reply by cok3307788** [reply]
希望有这个功能

**Reply by canyie** [reply]
做不到

**Reply by cok3307788** [reply]
> 做不到

请问为什么做不到，是不是因为有TEE的原因？
